back/engines/commercial/analysis/spec/acceptance/tags_spec.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Tags' do
+  explanation 'Analysis tags'
+
+  before { header 'Content-Type', 'application/json' }
+
+  let(:json_response) { json_parse(response_body) }
```

Comment:
we have a few helper methods in `api_helper.rb` that make this redundant. You can use `json_response_body` everywhere where you used `json_response`. Okay to leave it too, fyi.